helm/bold-common/README.md

```diff
@@ -91,31 +90,47 @@ For Helm chart, you'll need to craft a `values.yaml`.
        persistentVolume*
       </td>
       <td>
-       Please refer to <a href='/helm/docs/configuration.md#persistent-volume'>this</a> section to know more on how to set Persistant Volumes for Bold BI.
+       Please refer to <a href='/helm/docs/common-configuration.md#persistent-volume'>this</a> section to know more on how to set Persistant Volumes for Bold BI.
+      </td>
+    </tr>
+        <tr>
+      <td>
+       image *
+      </td>
+      <td>     
+       By default latest image tag is used in Helm chart. Please refer to <a href='/helm/docs/common-configuration.md#image'>this</a> section for configure a image tag for Bold BI, Bold Reports and ID. 
+      </td>
+    </tr>
+    <tr>
+      <td>
+       versions *
+      </td>
+      <td>
+       By default latest version is used in Helm chart. Please refer to <a href='/helm/docs/common-configuration.md#version'>this</a> section for configure a version for Bold BI, Bold Reports and IDP version 
       </td>
     </tr>
     <tr>
       <td>
        loadBalancer
       </td>
       <td>
-       Currently we have provided support for Nginx and Istio as Load Balancers in Bold BI. Please refer to <a href='/helm/docs/configuration.md#load-balancing'>this</a> section for configuring Load balancer for Bold BI.
+       Currently we have provided support for Nginx and Istio as Load Balancers in Bold BI. Please refer to <a href='/helm/docs/common-configuration.md#load-balancing'>this</a> section for configuring Load balancer for Bold BI.
```

Comment:
Currently, we have provided support for Nginx and Istio as Load Balancers in Bold BI. Please refer to <a href='/helm/docs/common-configuration.md#load-balancing'>this</a> section for configuring Load balancer for Bold BI.

Reply:
concern addressed